tsfile/src/test/java/org/apache/iotdb/tsfile/write/writer/ForceAppendTsFileWriterTest.java

```diff
@@ -56,6 +65,38 @@ public void test() throws Exception {
     writer.close();
     ForceAppendTsFileWriter fwriter = new ForceAppendTsFileWriter(file);
     assertEquals(firstMetadataPosition, fwriter.getTruncatePosition());
+
+    // write more data into this TsFile
+    writer = new TsFileWriter(fwriter);
+    writer.registerTimeseries(new Path("d1.s1"),
+        new MeasurementSchema("s1", TSDataType.FLOAT, TSEncoding.RLE));
+    writer.registerTimeseries(new Path("d1.s2"),
+        new MeasurementSchema("s2", TSDataType.FLOAT, TSEncoding.RLE));
```

Comment:
Please note, since we aren't able to recover schema from metadata, I register the timeseries again here.